Breadcrumbs.py

```diff
@@ -108,4 +119,182 @@ def get_points_by_row(row):
+
+
+class BreadcrumbsPhantomCommand(sublime_plugin.TextCommand):
+
+  def __init__(self, view):
+    self.phantoms_visible = False
+    self.view = view
+    self.phantom_set = sublime.PhantomSet(view, 'breadcrumbs')
+
+  def close_phantoms(self, href):
+    self.view.erase_phantoms('breadcrumbs')
+    self.phantoms_visible = False
+
+  def run(self, edit):
+    if self.phantoms_visible:
+      self.close_phantoms('close')
+      return
+    else:
+      self.close_phantoms('close')
+
+    stylesheet = '''
+      <style>
+        html {
+          --base-bg: color(var(--bluish) blend(var(--background) 30%));
+          --accent-bg: color(var(--base-bg) blend(var(--foreground) 90%));
+        }
+        div.phantom-arrow {
+          border-top: 0.4rem solid transparent;
+          border-left: 0.5rem solid var(--base-bg);
+          width: 0;
+          height: 0;
+        }
+        div.phantom {
+          margin: 0 0 0.2rem;
+          border-radius: 0 0.2rem 0.2rem 0.2rem;
+        }
+        div.phantom a {
+          text-decoration: inherit;
+        }
+       .crumb {
+          line-height: 2rem;
+          padding-right: 1rem;
+        }
+        .separator {
+          border: 1rem solid;
+          width:0;
+          height:0;
+          font-size:1rem;
+          line-height:0px;
+        }
+        .crumb-1,
+        .separator-1 {
+          background-color: var(--base-bg);
+        }
+        .crumb-2,
+        .separator-2 {
+          background-color: var(--accent-bg);
+        }
+        .separator-1 {
+          border-color: var(--base-bg);
+          border-left-color: var(--accent-bg);
+        }
+        .separator-2 {
+          border-color: var(--accent-bg);
+          border-left-color: var(--base-bg);
+        }
+        div.phantom a.close {
+          vertical-align: middle;
+          line-height: 2rem;
+          padding: 0 0.7rem 0 0.8rem;
+          border-radius: 0 0.2rem 0.2rem 0;
+          font-weight: bold;
+          background-color: var(--base-bg);
+        }
+      </style>
+    '''
+
+    template = '''
+      <body id="inline-breadcrumbs">
+        {stylesheet}
+        <div class="phantom-arrow"></div>
+        <div class="phantom">{breadcrumbs}<a class="close" href="close">''' + chr(0x00D7) + '''</a></div>
+      </body>
+    '''
+
+    phantoms = []
+
+    for region in self.view.sel():
+      (row, col) = self.view.rowcol(region.begin())
+
+      crumb_elements = []
+      for i,crumb in enumerate(make_breadcrumbs(self.view, row, False)):
+        parity = (i % 2) + 1
+        crumb_elements.append('<span class="separator separator-{parity}"> </span><span class="crumb crumb-{parity}">'.format(parity = parity) + html.escape(crumb, quote=False) + '</span>')
+
+      body = template.format(
+          breadcrumbs=''.join(crumb_elements),
+          stylesheet=stylesheet
+      )
+      phantom = sublime.Phantom(
+          region,
+          body,
+          sublime.LAYOUT_BELOW,
```

Comment:
So, yeah, I can go either way on this. You want to have the phantoms as LAYOUT_BLOCK?